public/scripts/worktrollyUI.js

```diff
@@ -1,43 +1,67 @@
-$("#createNewTaskButton").click(function() {
+$("#createNewTaskButton").click(function () {
     newPage = "createNewTask";
     uiLoader();
     var date = new Date();
     $("#creationDateCreateNewTask").html(date.getDate() + "/" + (date.getMonth() + 1) + "/" + date.getFullYear());
 
 });
 
-$("#backToMainFromCreateNewTask").click(function() {
+$("#backToMainFromCreateNewTask").click(function () {
```

Comment:
Unnecessary changes here as well, Please use the correct formatter, talk to shivam regards to this.